src/components/FolderListItem/FolderListItem.js

```diff
@@ -0,0 +1,40 @@
+import styles from "./FolderListItem.module.css";
+
+function FolderListItem({ folders, selectedId, onSelectedFolder }) {
+  const handleClick = (e) => {
+    onSelectedFolder({ name: e.target.textContent, id: e.target.id });
+  };
+
+  return (
+    <div className={styles["wrapper"]}>
```

Comment:
특별한 이유가 없다면 일관성있게 사용하면 좋을 것 같아요!
```suggestion
    <div className={styles.wrapper}>
```